packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -911,9 +911,16 @@ async function handler(
   const calEventUserFieldsResponses =
     "calEventUserFieldsResponses" in reqBody ? reqBody.calEventUserFieldsResponses : null;
 
+  const createStaticTitle = () => {
+    return eventNameObject.eventType + ' by ' + eventNameObject.host
```

Comment:
I think the new logic should be inside of our core logic. I think for seated events the second attendee gets booking title from DB, when we should be reading event-type.title and parsing with getEventName()